deploy/crds/kustomize_configurationpolicy/policy.open-cluster-management.io_configurationpolicies.yaml

```diff
@@ -159,14 +173,16 @@ spec:
                       - mustonlyhave
                       type: string
                     objectDefinition:
-                      description: ObjectDefinition defines required fields for the
-                        object
+                      description: |-
+                        ObjectDefinition defines required fields, in whole or in part, to be compared with objects on
+                        the cluster.
                       type: object
                       x-kubernetes-preserve-unknown-fields: true
                     recordDiff:
                       description: |-
-                        RecordDiff specifies whether (and where) to log the diff between the object on the
-                        cluster and the objectDefinition in the policy. Defaults to "None".
+                        RecordDiff specifies whether (and where) to log the diff between the object on the cluster and
+                        the objectDefinition in the policy. Options are "Log" to log the diff in the
+                        config-policy-controller pod and "None" to not log the diff. The default value is "None".
```

Comment:
I've used "difference" here, the reason being that it could be logged when the remediation action is `inform` and the object is different but wouldn't be updated.